atomsci/ddm/utils/model_file_reader.py

```diff
@@ -69,8 +68,7 @@ def __init__(self, data_file_path):
                 try:
                     meta_info = tarball.getmember('./model_metadata.json')
                 except KeyError:
-                    print(f"{tarpath} is not an AMPL model tarball")
-                    return {}
+                    raise KeyError(f"{data_file_path} is not an AMPL model tarball")
```

Comment:
this would have thrown an error if model_metadata.json was missing; needs test

Reply:
I think this commit has a test for this branch. https://github.com/ATOMScience-org/AMPL/commit/e007f80e66884460776a77ec18809462e4b7eab2